src/components/NetworkLogo.tsx

```diff
@@ -1,12 +1,23 @@
 import config from '@/configuration';
+import { ChainType } from '@/types/config';
 
-const networkIds = Object.keys(config.NETWORKS_CONFIG).map(Number);
+const networkIds = Object.keys(config.EVM_NETWORKS_CONFIG).map(Number);
 
-const NetworkLogo = (props: { chainId?: number; logoSize?: number }) => {
-	const { chainId, logoSize } = props;
+interface INetworkLogoProps {
+	chainId?: number;
+	logoSize?: number;
+	chainType?: ChainType;
+}
+
+const NetworkLogo = (props: INetworkLogoProps) => {
+	const { chainId, logoSize, chainType } = props;
+	if (chainType && chainType !== ChainType.EVM) {
+		return config.NON_EVM_NETWORKS_CONFIG[chainType].chainLogo(logoSize);
+	}
 	if (chainId && networkIds.includes(chainId)) {
-		return config.NETWORKS_CONFIG[chainId].chainLogo(logoSize);
-	} else return null;
+		return config.EVM_NETWORKS_CONFIG[chainId].chainLogo(logoSize);
+	}
+	return null;
```

Comment:
It can be shorter by the same approach used in `getChainName` in the src/lib/network.ts file.

Reply:
Done!